portfolio/src/main/webapp/script.js

```diff
@@ -72,24 +72,44 @@ async function displayCommentsToPage() {
 
   const arrayTextElement = document.getElementById('array-text-container');
   arrayTextElement.innerHTML = '';
-    
-  for (let i = 0; i < textArray.length; i++) {
-    let comment = textArray[i];
-    let commentString = comment.comment;
-    let sentimentScore = comment.sentimentScore;
+  
+  if (!textArray.length) {
+    arrayTextElement.innerHTML = 'Be the first to comment below!';
+  }
 
-    arrayTextElement.appendChild(
-        createListElement(commentString + ' ' + '(' + sentimentScore + ')' + '\n'));
+  for (let i = 0; i < textArray.length; i++) {
+    createCommentElement(textArray[i], i);
   }
 }
 
-/*
- * Creates an <li> element containing text. 
+/**
+ * Creates comment element containing text, name, and image based on sentiment.
  */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+function createCommentElement(comment, i) {
+  let commentString = comment.comment;
+  let userName = comment.userName;
+  let sentimentScore = comment.sentimentScore;
```

Comment:
Wanna see something cool?

```suggestion
  const {comment, userName, sentimentScore} = comment;
```

It's called Destructuring Assignment https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Super cool! Thank you!